draft-ietf-rats-architecture.md

```diff
@@ -273,12 +273,24 @@ plans to support in the TEEP architecture {{?I-D.ietf-teep-architecture}}.
 
 # Two Types of Environments
```

Comment:
something like... "of an Attester" only better is required here? I am aware that the title is still a placeholder and very literal, but I had a hard time coming up with a more appropriate one. Separate issue maybe?

Reply:
I agree with adding "of an Attester", good suggestion